openapi/scim/scim.yaml

```diff
@@ -678,16 +680,18 @@ components:
           example: Hollywood
         region:
           type: string
-          example: CA
+          format: countrycode-statecode
+          example: CA-SK
         postalCode:
           type: string
           example: '91608'
         country:
           type: string
-          example: USA
+          format : ISO 3166-1 alpha-2 {two letters}
```

Comment:
We should move the contents of the format field to the description. According the the [open api specification](https://spec.openapis.org/oas/v3.1.0#data-types) the `format` field should be one of the values [defined by the JSON Schema standard](https://datatracker.ietf.org/doc/html/draft-bhutton-json-schema-validation-00#section-7.3). The values basically map to well known regex patterns so that it can be automatically validated.

For the platform users resouce we set a pattern and min/max length instead.
<img width="412" alt="image" src="https://user-images.githubusercontent.com/6698651/220673018-b4c45934-5feb-4c2a-ac0e-a8cbc26b999d.png">